drivers/sensor/tmp108/tmp108.c

```diff
@@ -202,13 +199,18 @@ static int tmp108_attr_set(const struct device *dev,
 	uint16_t mode = 0;
 	uint16_t reg_value = 0;
 	int result = 0;
+	int32_t uval;
 
 	if (chan != SENSOR_CHAN_AMBIENT_TEMP && chan != SENSOR_CHAN_ALL) {
 		return -ENOTSUP;
 	}
 
 	switch ((int) attr) {
 	case SENSOR_ATTR_HYSTERESIS:
+#if DT_PROP(DT_INST(0, ti_tmp108), variant_as621x)
+		LOG_WRN("AS621x Series lacks Hysterisis setttings");
+		break;
+#endif
```

Comment:
This won't work if there are multiple instances of the driver. Add an element to `struct tmp108_config` so you can look up the value on a per-instance basis.

Reply:
Thanks @MaureenHelm will add this

Reply:
@MaureenHelm I couldn't really think of an easy way to do this, so I just ended up requiring the user to configure this in the Kconfig by indicating whether they are compiling for a board with the as621 or the tmp108. At this point it would be a lot of work to allow both chipsets to operate on the same board using the same driver, so I think this is a good compromise.